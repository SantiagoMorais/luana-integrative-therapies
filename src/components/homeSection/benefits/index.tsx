import { fontSize, fontWeight, theme } from "@styles/theme"
import styled from "styled-components"
import data from "@json/index.json"

export const Benefits = () => {
    return (
        <Container>
            <div className="title">Benef√≠cio das Terapias Integrativas</div>
            <ul className="benefits">
                {data &&
                    <>
                        {data.benefits.map((benefit, index) =>
                            <li key={index} className="benefit">
                                <p>{index + 1}.</p>
                                <p>{benefit}</p>
                            </li>
                        )}
                    </>
                }
            </ul>
        </Container>
    )
}

const Container = styled.div`
    width: 100%;
    background-color: ${theme.primaryColor};
    padding: 3rem 2rem 0;
    display: flex;
    flex-direction: column;
    align-items: center;

    .title {
        width: 100%;
        font-size: ${fontSize.extraLargeSize};
        text-align: center;
        padding: 1rem;
    }

    .benefits {
        width: 100%;
        display: flex;
        flex-wrap: wrap;
        justify-content: center;
        gap: 1rem;
        padding: 0 1rem 3rem 1rem;
        max-width: 144rem;

        .benefit {
            font-size: ${fontSize.mediumSize};
            text-align: center;
            flex-grow: 1;
            flex-basis: 20rem;
            padding: 1rem;
            font-weight: ${fontWeight.thin};
            max-width: 40rem;

            p {
                &:first-child {
                    font-size: 150%;
                    font-weight: ${fontWeight.thin};
                    padding-bottom: 1rem;
                    border-bottom: .2rem solid ${theme.secondaryColor};
                    margin-bottom: 1rem;
                }
            }
        }
    }

    @media (max-width: 725px) {
        padding: 2rem 1rem 0;
    
        .title {
            font-size: ${fontSize.largeSize};
        }
    
        .benefits {
            gap: 1rem;
    
            .benefit {
                font-size: ${fontSize.basicSize};
            }
        }
    }

    @media (max-width: 425px) {
        .title {
            font-size: ${fontSize.mediumSize};
        }
    
        .benefits {
            gap: 0rem;
    
            .benefit {
                font-size: ${fontSize.smallSize};
            }
        }
    }
`