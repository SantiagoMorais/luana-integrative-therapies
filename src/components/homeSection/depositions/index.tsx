import styled from "styled-components";
import data from "@json/index.json";
import { fontSize, fontWeight, ITheme } from "@styles/theme";
import { DepositionsCarousel } from "./depositionsCarousel";
import { useThemeContext } from "hooks/useThemeContext";

export const Depositions = () => {
   const theme = useThemeContext();

   const getTopicsInfo = () => {
      interface IDepositions {
         id: string;
         deposition: string;
         name: string;
      }

      const topicInfo: IDepositions[] = data.depositions.map((deposition) => {
         return {
            id: deposition.patientName,
            deposition: deposition.deposition,
            name: deposition.patientName,
         };
      });

      return topicInfo;
   };

   return (
      <Container $theme={theme}>
         <h2 className="title">Depoimentos de pacientes</h2>
         <div className="content">
            <div className="depositions">
               <DepositionsCarousel info={getTopicsInfo()} slidesNumber={2} />
            </div>
         </div>
      </Container>
   );
};

const Container = styled.div<{ $theme: ITheme }>`
   margin-bottom: 2rem;
   display: flex;
   flex-direction: column;
   align-items: center;

   .title {
      width: 100%;
      text-align: center;
      font-size: ${fontSize.extraLargeSize};
      font-weight: ${fontWeight.medium};
      background-color: ${({ $theme }) => $theme.primaryColor};
      padding: 2rem 0;
   }

   .content {
      display: flex;
      justify-content: center;
      max-width: 144rem;
      width: 100%;

      .depositions {
         width: 100%;
         padding: 2rem;
         font-size: ${fontSize.extraLargeSize};
      }
   }

   @media (max-width: 425px) {
      margin-bottom: 1rem;

      .title {
         font-size: ${fontSize.mediumSize};
      }
   }
`;
