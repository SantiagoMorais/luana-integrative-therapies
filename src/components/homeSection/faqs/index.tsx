import styled from "styled-components";
import data from "@json/index.json";
import { fontSize, fontWeight, theme } from "@styles/theme";
import { useState } from "react";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
import { Collapse } from "react-collapse";

export const FAQs = () => {
   const [questionSelected, setQuestionSelected] = useState("");

   const handleAccordion = (faq: string) => {
      questionSelected === faq
         ? setQuestionSelected("")
         : setQuestionSelected(faq);
   };

   return (
      <Container>
         <h2 className="title">Perguntas Frequentes</h2>
         {data.faqs && (
            <ul className="faqs">
               {data.faqs.map((faq, index) => (
                  <li
                     className="faq"
                     key={index}
                     onClick={() => handleAccordion(faq.question)}
                  >
                     <div className="questionBar" data-testid="question">
                        <p className="question">{faq.question}</p>
                        <FontAwesomeIcon
                           icon={faChevronDown}
                           className={`icon ${
                              questionSelected === faq.question && "rotate"
                           }`}
                        />
                     </div>
                     {/* <p
                        className={`answer ${
                           questionSelected === faq.question ? "selected" : ""
                        }`}
                        data-testid="answer"
                     >
                        {faq.answer}
                     </p> */}
                     <Collapse isOpened={questionSelected === faq.question}>
                        {faq.answer}
                     </Collapse>
                  </li>
               ))}
            </ul>
         )}
      </Container>
   );
};

const Container = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   gap: 3rem;
   margin-bottom: 5rem;
   padding: 0 3rem;

   .ReactCollapse--collapse {
      transition: height .5s ease;
      margin: 1rem 1rem 0;
      font-size: ${fontSize.smallSize};
      font-weight: ${fontWeight.medium};
   }

   .title {
      font-size: ${fontSize.extraLargeSize};
      font-weight: ${fontWeight.medium};
   }

   .faqs {
      max-width: 100rem;
      display: flex;
      flex-direction: column;
      gap: 2rem;
      font-size: ${fontSize.smallSize};

      .faq {
         cursor: pointer;

         &:hover > .question {
            opacity: 0.6;
         }

         .questionBar {
            font-size: ${fontSize.basicSize};
            padding-bottom: 1rem;
            border-bottom: 0.2rem solid ${theme.secondaryColor};
            transition: 0.3s;
            display: flex;
            align-items: center;
            justify-content: space-between;
            gap: 2rem;
            font-weight: ${fontWeight.medium};

            .question {
               width: 100%;
               text-align: left;
            }

            .icon {
               transition: 0.5s ease;

               &.rotate {
                  transform: rotate(180deg);
               }
            }
         }
      }
   }

   @media (max-width: 768px) {
      padding: 0 2rem;

      .title {
         font-size: ${fontSize.mediumSize};
      }

      .faqs {
         .faq {
            .questionBar {
               text-align: center;
               font-size: ${fontSize.smallSize};
            }
         }
      }
   }
`;
