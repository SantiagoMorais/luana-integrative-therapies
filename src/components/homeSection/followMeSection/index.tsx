import styled from "styled-components"
import followMeImage from "@assets/imgs/luanaImageWithoutBG-3.png"
import moonImage from "@assets/imgs/moonPNG.png"
import starsImage from "@assets/imgs/nightSkyPNG.jpg"
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
import { faInstagram } from "@fortawesome/free-brands-svg-icons"
import { faCalendarDays } from "@fortawesome/free-solid-svg-icons"
import { theme } from "@styles/theme"
import { instagramLink, whatsappLink } from "@utils/variables"

export const FollowMeSection = () => {
    return (
        <Container>
            <div className="content">
                <div className="moonContainer">
                    <div className="moonImageContainer">
                        <img src={moonImage} alt="imagem lua" className="moonImage" />
                    </div>
                </div>
                <div className="starsContainer">
                    <div className="starsImageContainer">
                        <img src={starsImage} alt="imagem estrelas" className="starsImage" />
                    </div>
                </div>
                <div className="info instagram">
                    <h3 className="title">
                        Gostaria de conhecer um pouco mais sobre o mundo das terapias integrativas?
                    </h3>
                    <p className="message">
                        Me siga no instagram para receber todas as novidades!
                    </p>
                    <button className="button">
                        <a href={instagramLink} className="link" target="_blank" >
                            <FontAwesomeIcon icon={faInstagram} className="icon" />
                            <p className="buttonText">Instagram</p>
                        </a>
                    </button>
                </div>
                <div className="imageContainer">
                    <img src={followMeImage} alt="Siga-me no Instagram" className="instragramImage" />
                </div>
                <div className="info">
                    <h3 className="title">
                        Agende agora sua consulta comigo pelo Whatsapp!
                    </h3>
                    <p className="message">
                        Precisa de orientações ou quer agendar uma consulta?
                        Entre em contato clicando no botão abaixo e faça sua reserva agora mesmo.
                    </p>
                    <button className="button">
                        <a href={whatsappLink} className="link" target="_blank" >
                            <FontAwesomeIcon icon={faCalendarDays} className="icon" />
                            <p className="buttonText">Agendar consulta</p>
                        </a>
                    </button>
                </div>
            </div>
        </Container>
    )
}

const Container = styled.section`
    margin-bottom: 2rem;
    display: flex;
    flex-direction: column;
    align-items: center;
    background: linear-gradient(90deg, ${theme.tertiaryColor} 0%, ${theme.primaryColor} 100%);

    .content {
        width: 100%;
        max-width: 144rem;
        display: flex;
        justify-content: center;
        padding: 0 2rem;
        gap: 2rem;
        align-items: center;
        position: relative;
        overflow-y: hidden;

        .moonContainer {
            position: absolute;
            top: 0;
            left: 0;
            width: 100%;
            height: 100%;
            overflow: hidden;
            
            .moonImageContainer {
                position: relative;
                width: 100%;
                height: 100%;
        
                .moonImage {
                    width: 60%;
                    height: 100%;
                    object-fit: cover;
                    object-position: -20rem bottom;
                    opacity: .6;
                    mask-image: radial-gradient(circle, rgba(0, 0, 0, 1) 50%, rgba(0, 0, 0, 0) 100%);
                    -webkit-mask-image: radial-gradient(circle, rgba(0, 0, 0, 1) 50%, rgba(0, 0, 0, 0) 100%);
                }
            }
        }

        .starsContainer {
            position: absolute;
            top: -5rem;
            right: 0;
            width: 70%;
            height: 100%;

            .starsImageContainer {
                .starsImage {
                    right: 0rem;
                    width: 100%;
                    height: auto;
                    object-fit: cover;
                    object-position:  center;
                    opacity: .7;
                    mask-image: radial-gradient(circle, black 10%, transparent 70%);
                    -webkit-mask-image: radial-gradient(circle, black 10%, transparent 70%);
                    transform: scale(-1);
                }
            }
        }
    
        .imageContainer {
            max-width: 40rem;
            flex: 2;
            position: relative;
            
            .instragramImage {
                width: 100%;
                height: 100%;
                scale: 1.3;
                object-fit: cover;
                object-position: center 5rem;
                filter: drop-shadow(0 0 .5rem white);
            }
        }

        .info {
            width: 100%;
            flex: 1;
            padding: 2rem 0;
            display: flex;
            flex-direction: column;
            gap: 1rem;
            z-index: 1;

            .title {
                font-size: 3rem;
                color: ${theme.textColor};
                font-weight: 600;
                text-align: center;
            }

            .message {
                font-size: 2.1rem;
                text-align: center;
            }

            .button {
                border: none;
                padding: .5rem 2rem;
                width: fit-content;
                align-self: center;
                border-radius: 1rem;
                background-color: ${theme.secondaryColor};
                font-size: 2.1rem;
                font-weight: 500;
                cursor: pointer;
                transition: .3s;
                opacity: .8;
                border: .2rem solid transparent;
                
                .link {
                    color: ${theme.secondaryTextColor};
                    display: flex;
                    align-items: center;
                    justify-content: center;
                    gap: 1rem;

                    .buttonText {
                        min-width: fit-content;
                    }
                }

                &:hover {
                    scale: 1.15;
                    opacity: 1;
                    border-color: ${theme.secondaryTextColor};
                }
            }

            &.instagram {
                text-align: end;
            }
        }
    }

    @media (max-width: 768px) {
        background: linear-gradient(0deg, ${theme.primaryColor} 0%, ${theme.tertiaryColor} 100%);

        .content {
            flex-direction: column;
            padding: 0 0 2rem;
            gap: 1rem;

            .moonContainer {
                width: 100%;
                
                .moonImageContainer {
                    .moonImage {
                        opacity: 1;
                        width: 100%;
                        height: 100%;
                        object-fit: cover;
                        object-position: center 50rem;
                    }
                }
            }

            .starsContainer {
                top: -5rem;
                width: 100%;
                height: 100%;

                .starsImageContainer {
                    .starsImage {
                        object-position: top;
                        opacity: 0.6;
                        mask-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0));
                        -webkit-mask-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0));
                    }

                }
            }
    
            .imageContainer {
                max-width: 100%;
                width: 100%;
                overflow: hidden;
                order: 1;
                flex: auto;
                padding: 2rem;

                .instragramImage {
                    scale: 1;
                    object-position: center;
                    mask-image: linear-gradient(to bottom, black, black 90%, transparent 100%);
                    -webkit-mask-image: linear-gradient(to bottom, black, black 90%, transparent 100%);
                }
            }
    
            .info {
                max-width: 100%;
                text-align: center;
                align-items: center;
                padding: 0 2rem;
                order: 2;

                .title {
                    text-align: center;
                }
            }
        }
    }
`