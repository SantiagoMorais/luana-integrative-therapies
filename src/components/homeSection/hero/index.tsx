import styled, { keyframes } from "styled-components"
import heroImage from "@assets/imgs/heroImageLargeScreenDevices.jpg"
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
import { faArrowDown } from "@fortawesome/free-solid-svg-icons"
import { fontWeight, theme } from "@styles/theme"

export const Hero = () => {
    return (
        <Container>
            <div className="slogan">
                <h2 className="title">Harmonizando Energia e Saúde</h2>
                <h3 className="subtitle">
                    Descubra uma jornada de cura e harmonia através de terapias integrativas.
                </h3>
            </div>
            <FontAwesomeIcon className="icon" icon={faArrowDown} data-testid="fontAwesomeIcon" />
        </Container>
    )
}

const animation = keyframes`
    0% {
        transform: translateY(0);
        opacity: .4;
    };
    50% {
        transform: translateY(1rem);
        opacity: 1;
    };
    100% {
        transform: translateY(0);
        opacity: .4;
    }
`

const Container = styled.div`
    flex-grow: 1;
    background: ${theme.secondaryTextColor} url(${heroImage}) no-repeat 15dvw 10%;
    background-size: cover;
    position: relative;
    border: solid ${theme.secondaryTextColor};
    border-width: .3rem 0;

    .slogan {
        position: absolute;
        width: 35%;
        left: 8%;
        top: 12%;
        color: ${theme.textColor};
        z-index: 1;
        display: flex;
        flex-direction: column;
        gap: 1rem;

        .title {
            font-size: 9rem;
            font-weight: ${fontWeight.bold};
            line-height: 90%;
            color: ${theme.textColor};
            text-transform: uppercase;
            filter: drop-shadow(0 0 .2rem #587aaa);
        }

        .subtitle {
            font-size: 5rem;
            font-weight: ${fontWeight.medium};
            color: ${theme.textColor};
            filter: drop-shadow(0 0 .2rem ${theme.secondaryColor});
            font-family: ${theme.fontFamily};
        }
    }

    .icon {
        font-size: 5rem;
        position: absolute;
        bottom: 3rem;
        right: 0;
        left: 0;
        margin: auto;
        animation: ${animation} 2s infinite;
        filter: drop-shadow(0 0 .5rem ${theme.secondaryTextColor});
        z-index: 2;
    }

    &::after, &::before {
        content: "";
        position: absolute;
        top: 0;
        bottom: 0;
        pointer-events: none;
    }

    &::after {
            left: 14.9dvw;
            width: 20dvw;
            background: linear-gradient(to left, rgba(255, 255, 255, 0), ${theme.secondaryTextColor});
    }

    &::before {
            left: 0;
            width: 15dvw;
            background: ${theme.secondaryTextColor};
        }

    @media (max-width: 1200px) {
        background-position: left 10%;

        .slogan {
            .title {
                font-size: 6rem;
            }

            .subtitle {
                font-size: 4rem;
            }
        }

        .icon {
            font-size: 4rem;
            bottom: 2.5rem;
        }
    }

    @media (max-width: 900px) {
        background-position: center 10%;

        .slogan {
            width: 80%;
            left: 0;
            right: 0;
            margin: auto;
            text-align: center;
            bottom: 5rem;
            top: auto;
            gap: 0;

            .title {
                font-size: 5rem;
            }

            .subtitle {
                font-size: 3.5rem;
            }
        }

        .icon {
            font-size: 3.5rem;
            bottom: 1.5rem;
        }

        &::after, &::before {
            left: 0;
            right: 0;
            top: auto;
            width: 100%;
        }

        &::before {
            bottom: 0;
            height: 8rem;
        }

        &::after {
            bottom: 7.9rem;
            height: 10rem;
            background: linear-gradient(to bottom, rgba(255, 255, 255, 0), ${theme.secondaryTextColor});
        }
    }

    @media (max-width: 768px) {
        .slogan {
            bottom: 4rem;
            .title {
                font-size: 4rem;
            }

            .subtitle {
                font-size: 2.6rem;
            }
        }
    
        .icon {
            font-size: 3rem;
        }
    }

    @media (max-width: 425px) {
        .slogan {
            width: 80%;
            
            .title {
                font-size: 3.2rem;
            }

            .subtitle {
                font-size: 2.1rem;
            }
        }

        .icon {
            bottom: 1rem;
        }
    }
`