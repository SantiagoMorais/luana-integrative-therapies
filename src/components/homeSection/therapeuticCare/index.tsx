import styled from "styled-components"
import luanaWithProducts from "@assets/imgs/imageToTherapeuticCare.jpg" 
import { fontSize, fontWeight } from "@styles/theme"

export const TherapeuticCare = () => {
    return (
        <Container>
            <div className="info">
                <h2 className="title">Como funcionam os atendimentos?</h2>
                <p className="functionality">
                    Você será atendido mediante a agendamento prévio, basta entrar em contato. As sessões têm em média 01:30hs de duração, e diante de uma anamnese integrativa escolhemos quais terapias vamos utilizar para as suas queixas. 
                    <br />
                    A duração do tratamento e espaço entre sessões dependerá das queixas e da resposta biológica. Já obtemos resultados positivos e de melhora desde a primeira sessão.
                    <br />
                    Caso seja uma preferência do paciente sentir os benefícios apenas de uma terapia, é possível agendar o horário específico, e a duração dependerá de qual prática será realizada. 
                </p>
            </div>
            <img className="image" src={luanaWithProducts} alt="Dra.Luana" />
        </Container>
    )
}

const Container = styled.div`
    display: flex;
    justify-content: center;
    padding: 8rem 3rem;
    gap: 3rem;
    flex-wrap: wrap-reverse;
    align-items: flex-end;

    .info, .image {
        flex: 1;
        max-width: 50rem;
        min-width: 30rem;
        width: 40%;
    }

    .info {
        font-size: ${fontSize.basicSize};
        font-weight: ${fontWeight.thin};

        .title {
            margin-bottom: 2rem;
            font-size: ${fontSize.extraLargeSize};
            font-weight: ${fontWeight.medium};
        }

        .functionality {
            text-align: justify;
        }
    }

    .image {
        max-height: 60rem;
        min-height: 40rem;
        object-fit: cover;
        border-radius: 1.5rem;
        box-shadow: .5rem .5rem 1rem rgba(0,0,0,.3);
    }

    @media (max-width: 768px) {
        padding: 3rem;
    }

    @media (max-width: 425px) {
        padding: 2rem;
        gap: 1rem;

        .info, .image {
            min-width: 20rem;
        }
        
        .info {
            font-size: ${fontSize.smallSize};
            font-weight: 400;
            
            .title {
                margin-bottom: 1rem;
                font-size: ${fontSize.mediumSize};
                text-align: center;
            }
        }
    }
`