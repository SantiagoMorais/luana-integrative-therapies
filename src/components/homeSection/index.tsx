import styled from "styled-components";

import { Benefits } from "./benefits";
import { TherapeuticCare } from "./therapeuticCare";
import { Depositions } from "./depositions";
import { FAQs } from "./faqs";
import { Hero } from "./hero";
import { FollowMeSection } from "./followMeSection";

import { Footer } from "@components/footer";
import { Header } from "@components/header";
import { useLocation } from "react-router-dom";
import { useEffect } from "react";

export const HomeSection = () => {
   const { pathname } = useLocation();

   useEffect(() => {
     window.scrollTo(0, 0);
   }, [pathname]);
   
   return (
      <Container>
         <div className="frontPage">
            <Header />
            <Hero />
         </div>
         <Benefits />
         <TherapeuticCare />
         <Depositions />
         <FollowMeSection />
         <FAQs />
         <Footer />
      </Container>
   );
};

const Container = styled.section`
   width: 100%;
   min-height: 100dvh;

   .frontPage {
      display: flex;
      flex-direction: column;
      flex: 1;
      height: 100dvh;
      max-height: 90dvh;
   }

   @media (max-width: 1920px) {
      .frontPage {
         max-height: 90rem;
      }
   }
`;
