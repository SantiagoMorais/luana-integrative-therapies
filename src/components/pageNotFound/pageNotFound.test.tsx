import { render, screen } from "@testing-library/react"
import correctImage from "@assets/imgs/Error404.jpg"
import { PageNotFound } from ".";
import { BrowserRouter, MemoryRouter, Route, Routes } from "react-router-dom";
import { HomeSection } from "@components/homeSection";
import userEvent from "@testing-library/user-event";

describe("<PageNotFound />", () => {
    it("should render the component correctly", () => {
        render(<PageNotFound />, {wrapper: BrowserRouter});
        const title = screen.getByRole("heading", {name: /Erro 404 - Página não encontrada/i});
        expect(title).toBeInTheDocument();
    })

    it("should render the image correctly", () => {
        render(<PageNotFound />, {wrapper: BrowserRouter});
        const image = screen.getByRole("img");
        expect(image).toHaveAttribute("src", correctImage);
    });

    it("should render the text correctly", () => {
        render(<PageNotFound />, {wrapper: BrowserRouter});
        const text = document.querySelectorAll(".text");
        expect(text).toMatchSnapshot();
    })

    it("should return to home page when clicked on the 'return' link", async () => {
        render(
            <MemoryRouter initialEntries={["/comming-soon"]}>
                <Routes>
                <Route path="*" element={<PageNotFound />}/>
                    <Route path="/" element={<HomeSection />}/>
                </Routes>
            </MemoryRouter>
        );
        const user = userEvent.setup();
        const returnLink = screen.getByRole("link", {name: /aqui/i})

        expect(returnLink).toBeInTheDocument();

        await user.click(returnLink);
        
        const slogan = screen.getByText(/Descubra uma jornada de cura e harmonia através de terapias integrativas./i);
        expect(slogan).toBeInTheDocument();
    })
})