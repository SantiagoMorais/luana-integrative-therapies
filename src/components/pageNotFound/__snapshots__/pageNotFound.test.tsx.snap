// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<PageNotFound /> > should render the text correctly 1`] = `NodeList []`;
